docs/pages/color_examples.js

```diff
@@ -0,0 +1,388 @@
+      <MainSection name="Color best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use colors to support creating distinction between elements, such as define primary and secondary actions. See [color usage](/color_usage) for reference and appropriate tokens."
+            defaultCode={`
+            <Flex gap={4}>
+              <IconButton
+                icon="speech"
+                iconColor="darkGray"
+              />
+              <Button
+                color="gray"
+                text="Visit"
+              />
+              <Button
+                color="red"
+                text="Save"
+              />
+              <IconButton
+                icon="share"
+                iconColor="darkGray"
+              />
+            </Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use color as a sole indicator of information. Color-only changes do not work well for those who may be color blind or have low vision; always supply an icon or text label for context."
+            defaultCode={`
+            <Flex direction="column" gap={2}>
+              <Label htmlFor={"solo-color"}><Text>Audience 1</Text></Label>
+              <Flex alignItems="center" gap={2}>
+                <Box rounding="circle" color="infoBase" width={12} height={12}/>
+                <SelectList
+                  id={"solo-color"}
+                  onChange={() => {}}
+                  options={[
+                    {label: 'Your total audience', value: '5'},
+                    {label: 'Active in the last week', value: '7'},
+                    {label: 'Active in the last month', value: '30'},
+                  ]}
+                />
+              </Flex>
+            </Flex>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use colors purposefully as it can convey meaning in multiple ways. Our extended color palette is used for communicating status or enhancing illustrations when needed. See [color usage](/color_usage) for reference."
+            defaultCode={`
+            <Flex direction="column" gap={3}>
+              <Badge type="info" text="Info"/>
+              <Badge type="success" text="Success" />
+              <Badge type="warning" text="Warning"/>
+              <Badge type="error" text="Error" />
+              <Badge type="neutral" text="Neutral" />
+            </Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Repurpose colors. Using colors for their intended meaning supports good comprehension and avoids usability and accessibility issues."
+            defaultCode={`
+            <Flex>
```

Comment:
can we add `gap={2}` 
![Screen Shot 2022-04-14 at 1 05 34 PM](https://user-images.githubusercontent.com/10593890/163438606-dfe631fa-1651-42ff-b2e0-c0b61d97442a.png)
?